Functions/Verbs/corefonts/script.js

```diff
@@ -0,0 +1,176 @@
+
+    new Resource()
+        .wizard(this._wizard)
+        .url("ftp://ftp.fi.debian.org/gentoo/distfiles/impact32.exe")
+        .checksum("86b34d650cfbbe5d3512d49d2545f7509a55aad2")
+        .name("impact32.exe")
+        .get(),
+
+    new Resource()
+        .wizard(this._wizard)
+        .url("ftp://ftp.fi.debian.org/gentoo/distfiles/times32.exe")
+        .checksum("20b79e65cdef4e2d7195f84da202499e3aa83060")
+        .name("times32.exe")
+        .get(),
+
+    new Resource()
+        .wizard(this._wizard)
+        .url("ftp://ftp.fi.debian.org/gentoo/distfiles/trebuc32.exe")
+        .checksum("50aab0988423efcc9cf21fac7d64d534d6d0a34a")
+        .name("trebuc32.exe")
+        .get(),
+
+    new Resource()
+        .wizard(this._wizard)
+        .url("ftp://ftp.fi.debian.org/gentoo/distfiles/verdan32.exe")
+        .checksum("f5b93cedf500edc67502f116578123618c64a42a")
+        .name("verdan32.exe")
+        .get(),
+
+    new Resource()
+        .wizard(this._wizard)
+        .url("ftp://ftp.fi.debian.org/gentoo/distfiles/webdin32.exe")
+        .checksum("2fb4a42c53e50bc70707a7b3c57baf62ba58398f")
+        .name("webdin32.exe")
+        .get()
+    ];
+
+    var numFonts = myStringArray.length;
+    for (var i = 0; i < numFonts; i++) {
+        new CabExtract()
+            .archive(fontResources[i])
+            .wizard(this._wizard)
+            .to(this.fontDirectory())
+            .extract();
+    }
+
+    this.registerFont()
+        .set("Arial", "Arial.TTF")
+        .do();
+    this.registerFont()
+        .set("Arial Bold", "Arialbd.TTF")
+        .do();
+    this.registerFont()
+        .set("Arial Bold Italic", "Arialbi.TTF")
+        .do();
+    this.registerFont()
+        .set("Arial Italic", "Ariali.TTF")
+        .do();
+    this.registerFont()
+        .set("Arial Black", "AriBlk.TTF")
+        .do();
+    this.registerFont()
+        .set("Comic Sans MS", "Comic.TTF")
+        .do();
+    this.registerFont()
+        .set("Comic Sans MS Bold", "Comicbd.TTF")
+        .do();
+    this.registerFont()
+        .set("Courier New", "Cour.TTF")
+        .do();
+    this.registerFont()
+        .set("Courier New Bold", "CourBD.TTF")
+        .do();
+    this.registerFont()
+        .set("Courier New Bold Italic", "CourBI.TTF")
+        .do();
+    this.registerFont()
+        .set("Courier New Italic", "Couri.TTF")
+        .do();
+    this.registerFont()
+        .set("Georgia", "Georgia.TTF")
+        .do();
+    this.registerFont()
+        .set("Georgia Bold", "Georgiab.TTF")
+        .do();
+    this.registerFont()
+        .set("Georgia Bold Italic", "Georgiaz.TTF")
+        .do();
+    this.registerFont()
+        .set("Georgia Italic", "Georgiai.TTF")
+        .do();
+    this.registerFont()
+        .set("Impact", "Impact.TTF")
+        .do();
+    this.registerFont()
+        .set("Times New Roman", "Times.TTF")
+        .do();
+    this.registerFont()
+        .set("Times New Roman Bold", "Timesbd.TTF")
```

Comment:
You can chain the .set directives to avoid creating a lot of objects and running many regedit processes